### DISCUSSION TITLE
Custom Landscape

### DISCUSSION THREAD

**Discussion by Jabilbao** [Stellarium/stellarium#2608] [Q&A]
Ok, here goes my first question for the group. 
I have created my panorama in Photoshop (PS) and edited as per many YouTube videos. Also created my text file which I have edited to a ".ini" file extension. I have verified the accuracy of the ini file and that it refers to the .PNG file created by PS. I have Zipped both files together.
When trying to load in to Stellarium within  Landscape, it gives me that it's either a non Zip or it does not contain a landscape error.
I have checked other reports and have made sure that the extensions do not appear to be repeated (.txt.txt or png.png) as I have "see extensions" turned on . 
I have tried using the files on my Mac and PC and I get the same error. 
I have attached my files as a ".txt" as this reporting does not accept a ".ini" file upload. Both .PNG and .TXT files are attached.
What am I missing?.
Thanks for your help.

[Landscape.txt](https://github.com/Stellarium/stellarium/files/9447038/Landscape.txt)
![landscape](https://user-images.githubusercontent.com/112422240/187271847-6c9d025f-f8a4-4a23-9016-47c683724361.png)

**Comment by gzotti** [comment]
Please see the official documentation in the User Guide (chapter 7), or see examples from the installed program. Your landscape.ini is at least missing the [landscape] and [location] headers.
BTW you don't have to ZIP the files. Just make a folder in your User Data Directory. (See chapter 5 for location)

**Reply by Jabilbao** [reply]
Thank you. I finally got it to work. Had some issues preparing the files on a Mac but all done now. I do have to improve my panorama and angle view but its working. It did ask for a Zip file though.
Thank you again for your quick assistance. Its very much appreciated.
